apis/capabilities/v1beta1/product_types.go

```diff
@@ -353,7 +353,12 @@ type OIDCSpec struct {
 	IssuerType string `json:"issuerType"`
 
 	// Issuer is the OIDC issuer
-	IssuerEndpoint string `json:"issuerEndpoint"`
+	// +optional
+	IssuerEndpoint string `json:"issuerEndpoint,omitempty"`
```

Comment:
Seeing this when creating your verification CR 
`Missing property "issuerEndpoint".` I don't think its a problem :thinking: . Just surprised that the UI is flagging it

Reply:
`Error "Required value" for field "spec.oidc.issuerEndpoint".` Guess I was wrong. `issuerEndpoint: ""`

Reply:
Seeing other issue, 
```yaml
      message: >-
        spec.openapiRef: Invalid value:
        v1beta1.OpenAPIRefSpec{SecretRef:(*v1.ObjectReference)(nil),
        URL:(*string)(0xc001075990)}: Missing OIDC definitions in CR. The
        referenced OpenAPI spec's sec scheme is openIdConnect or oauth2, the
        spec.oidc must not be nil or empty
      status: 'True'
      type: Failed
    - lastTransitionTime: '2023-08-25T11:53:15Z'
      message: >-
        spec.openapiRef: Invalid value:
        v1beta1.OpenAPIRefSpec{SecretRef:(*v1.ObjectReference)(nil),
        URL:(*string)(0xc001075990)}: Missing OIDC definitions in CR. The
        referenced OpenAPI spec's sec scheme is openIdConnect or oauth2, the
        spec.oidc must not be nil or empty
``` I will sync with you on monday.

Reply:
Taking another look at this , can see the field is optional so shouldn't be failing on this . Guessing there was some stale CRD's on the cluster.

Reply:
@austincunningham  thank you for review and notes. I did changes now. It's on latest commit. Could you please look. I don't see this error now, but some other, Warning, that related to loading of swagger file, but all seems working fine. Pets Product created on 3scale.  Also - OpenID Connect Issuer filed set correctly, as in secret, and authorization flow settings taken from CR. All looks good, I will recheck swagger itself. Log attached. Thank you
[openapi-example_log.txt](https://github.com/3scale/3scale-operator/files/12520199/openapi-example_log.txt)